src/form/grid/index.tsx

```diff
@@ -74,14 +85,39 @@ const Subgrid = ({
 };
 
 const getCellStyle = (cell: any) => {
-  const applyStyles = new ApplyStyles(cell, ['cell']);
+  const applyStyles = new ApplyStyles(cell, [
+    'cell',
+    'cellHover',
+    'cellActive'
+  ]);
   applyStyles.applyBorders('cell');
   applyStyles.applyCorners('cell');
   applyStyles.applyBackgroundImageStyles('cell');
   applyStyles.apply('cell', 'background_color', (c: any) => ({
     backgroundColor: `#${c}`
   }));
-  return applyStyles.getTarget('cell');
+  applyStyles.apply('cellHover', 'background_color', (a: any) => {
+    const color = `${adjustColor(a || 'ffffffff', -20)}!important`;
+    return {
+      backgroundColor: color,
+      borderColor: color,
+      transition: 'background 0.3s !important'
```

Comment:
maybe this transition style can be applied statically to the cell? applyStyles is only necessary for dynamic styles that may vary between mobile and desktop

Reply:
Just removing it